packages/fxa-payments-server/src/components/Profile.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Profile } from './Profile';
+
+storiesOf('components/Profile', module)
+  .add('with all profile data', () => (
```

Comment:
This looks pretty unwrapped:

![image](https://user-images.githubusercontent.com/44390/58832602-e7ec4600-8614-11e9-97dc-1a85f0958733.png)

For Storybook, should we make sure it's inside a representative container?

Reply:
Sure, though we don't have one implemented yet.